float64 x
float64 y
float64 z
int32 id
float64 gain
float64 yaw
